From 9928ce85961d70db3f4ab795ea9368c126e55278 Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Wed, 14 Feb 2024 09:33:56 +0100
Subject: [PATCH 082/120] stm32mp2: update part number for STM32MP251/3

update part number for STM32MP251/3.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/u-boot.git 5c1019d520f286fa6e3fb9bd933e38197f7a382e stm32mp2: update part number for STM32MP251/3]
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Change-Id: Ib06ebce8a04f48a0228196f69c1938ec0769a0fd
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/u-boot/+/359937
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm/mach-stm32mp/include/mach/sys_proto.h | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/mach-stm32mp/include/mach/sys_proto.h b/arch/arm/mach-stm32mp/include/mach/sys_proto.h
index 19073668497..a9613a681be 100644
--- a/arch/arm/mach-stm32mp/include/mach/sys_proto.h
+++ b/arch/arm/mach-stm32mp/include/mach/sys_proto.h
@@ -33,20 +33,20 @@
 /* ID for STM32MP25x = Device Part Number (RPN) (bit31:0) */
 #define CPU_STM32MP257Cxx       0x00002000
 #define CPU_STM32MP255Cxx       0x00082000
-#define CPU_STM32MP253Cxx       0x000B2004
-#define CPU_STM32MP251Cxx       0x000B3065
+#define CPU_STM32MP253Cxx       0x000B300C
+#define CPU_STM32MP251Cxx       0x000B306D
 #define CPU_STM32MP257Axx       0x40002E00
 #define CPU_STM32MP255Axx       0x40082E00
-#define CPU_STM32MP253Axx       0x400B2E04
-#define CPU_STM32MP251Axx       0x400B3E65
+#define CPU_STM32MP253Axx       0x400B3E0C
+#define CPU_STM32MP251Axx       0x400B3E6D
 #define CPU_STM32MP257Fxx       0x80002000
 #define CPU_STM32MP255Fxx       0x80082000
-#define CPU_STM32MP253Fxx       0x800B2004
-#define CPU_STM32MP251Fxx       0x800B3065
+#define CPU_STM32MP253Fxx       0x800B300C
+#define CPU_STM32MP251Fxx       0x800B306D
 #define CPU_STM32MP257Dxx       0xC0002E00
 #define CPU_STM32MP255Dxx       0xC0082E00
-#define CPU_STM32MP253Dxx       0xC00B2E04
-#define CPU_STM32MP251Dxx       0xC00B3E65
+#define CPU_STM32MP253Dxx       0xC00B3E0C
+#define CPU_STM32MP251Dxx       0xC00B3E6D
 
 /* return CPU_STMP32MP...Xxx constants */
 u32 get_cpu_type(void);
-- 
2.47.2

